spec/models/order_spec.rb

```diff
@@ -12,4 +12,21 @@
     it {should belong_to :user}
   end
 
+  describe "instance methods" do
+    before(:each) do
+      @user = create(:user)
+      @merchant = create(:merchant)
+      @order_1 = create(:order, user: @user)
+      @item_1 = create(:item, user: @merchant, price: 4.50)
+      @item_2 = create(:item, user: @merchant, price: 2.33)
+      @oi_1 = create(:order_item, item: @item_1, order: @order_1, quantity: 3, price_per_item: @item_1.price)
+      @oi_2 = create(:order_item, item: @item_2, order: @order_1, quantity: 1, price_per_item: @item_2.price - 0.33)
+    end
+
+    it "#grand_total" do
+      total_cost = @oi_1.price_per_item * @oi_1.quantity + @oi_2.price_per_item * @oi_2.quantity
+
+      expect(@order_1.grand_total).to eq(total_cost)
```

Comment:
Update with:
expect(number_to_currency(@order_1.grand_total)).to eq(number_to_currency(total_cost))

Reply:
I'm not going to make this change, because we're in the model. We shouldn't be formatting things here -- we should just leave the raw calculated number.

Reply:
Ha, good point!